### ISSUE TITLE
Add tabular view option to dashboard search results

### ISSUE THREAD

**Issue by dskaggs** [AnalogJ/lodestone#32]
On the main dashboard, it would be nice to have the option to view the search results in a tabular format instead of the default card view. This will help more easily deal with large numbers of results as our document libraries go.

We can add a control to the result header to switch between tabular and card view.

**Comment by AnalogJ** [comment]
Definitely! I have almost 100,000 documents that I need to load, so having an option for condensing the UI and making it more skimmable is definitely needed.
There's alot of backed stuff that I'm focused on right now, but I'll get to this before v1 is released.


<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by dskaggs** [comment]
I'd be happy to take a stab at it once you get the web container refactor completed and merged.
